AK/Format.cpp

```diff
@@ -567,7 +570,13 @@ ErrorOr<void> FormatBuilder::put_hexdump(ReadonlyBytes bytes, size_t width, char
                 TRY(put_literal("\n"sv));
             }
         }
-        TRY(put_u64(bytes[i], 16, false, false, true, Align::Right, 2));
```

Comment:
Not sure why adding specialised logic is desirable? this isn't meant to go fast, so I don't see why "total overkill" is a problem - less code, less logic, fewer issues